packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -70,6 +70,75 @@ const DEFAULT_SCHEDULE = {
   timezone: moment.tz.guess(),
 }
 
+const ReportReceivers = decorate([
+  provideState({
+    initialState: () => ({
+      receiver: '',
+    }),
+    effects: {
+      linkState,
+      addReceiver() {
+        const { receiver } = this.state
+        if (receiver === '') {
+          return
+        }
+        this.props.addReceiver(receiver)
+        this.resetState()
+      },
+      onKeyDown({ addReceiver }, event) {
+        if (event.keyCode === 13) {
+          event.preventDefault()
+          addReceiver()
+        }
+      },
+    },
+    computed: {
+      inputId: generateId,
+    },
+  }),
+  injectState,
+  ({ deleteReceiver, effects, receivers, state }) => (
+    <div>
+      <FormGroup>
+        <label htmlFor={state.inputId}>
+          <strong>{_('reportReceivers')}</strong>
+        </label>
+        <div className='input-group'>
+          <Input
+            id={state.inputId}
+            name='receiver'
+            onChange={effects.linkState}
+            onKeyDown={effects.onKeyDown}
+            value={state.receiver}
+          />
+          <span className='input-group-btn'>
+            <ActionButton
+              btnStyle='primary'
+              handler={effects.addReceiver}
+              icon='add'
+            />
+          </span>
+        </div>
+      </FormGroup>
+      <Ul className='mb-1'>
+        {map(receivers, (receiver, key) => (
+          <Li key={key}>
```

Comment:
If you remove an item from the array, all the items after it will jump left in the array. So each `key` will be associated with a different item from the previous render. So the index of the array is not a good `key` as it doesn't *identify* the item. `receiver` does, though.